free-3.md

```diff
@@ -10,41 +10,104 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The need met by my idea is a need for a simple beat creation for many music genres.
+I am trying to help those who casually make music and want an easy way to translate their
+production ideas into an actual beat/background music. Currently, these user would either
+have to learn more complicated sound design software or use things similar to the "App" 
+shown in class where you draw around on a grid. Idealy, if I could help them, I would
+make the beat generation simpler by allowing for the mouse clicks to be the main beat creator.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Most programs for sound/beat design are very complicated. I think a main reason
+for this is because with a program it is easy to add a ton of features and then
```

Comment:
I think it's really important that you mentioned this for the music beat creation. When writing up my ideas, I definitely wanted to add more and more features, but using a language is a great way to simplify the process of using it to something that's really manageable. If you want to get more complicated, as you mentioned, there are plenty of programs for that but for something simple this is a very important point.